docs/reference/experiment-config-reference.rst

```diff
@@ -319,25 +319,6 @@ While debugging, the logger will display lines highlighted in blue for easy iden
  Validation Policy
 *******************
 
-.. _experiment-config-min-validation-period:
```

Comment:
uh, I should probably tell you that @tara-det-ai told me we should mark it as deprecated but not delete it from the docs.

This is contrary to my preference, which is to let users look in old docs if they want to use old features.

But also it makes sense in this case because we don't have any deprecation warnings anywhere in the system for these fields; users would only know by looking at the docs.

Also this means we need to revert some of my docs deletions from the searcher-context-removal branch, which I realize now that I forgot to do :disappointed:.  It would be the removals from this file, I think.

That said, the other places where you remove things like "use min_validation_period to control validations" should be removed.  But the actual reference here should not.

Reply:
welp. ok then. i added them all back.

Reply:
thanks, sorry.